SS1-Default_Public_Segment.md

```diff
@@ -0,0 +1,92 @@
+# SS1: Default Public Segment
+
+Public Segments were added to Screeps to provide a way for different players to communicate with each other. Each player can specify any of their segments as public as well as one segment as their default public segment.
+
+In order for segments to be useful there needs to be an agreed upon protocol between different parties. This standard seeks to define a discovery protocol specifically for the default public segment which will allow players to advertise which protocols they support and how that data can be reached.
+
+
+## Goals
+
+* Enable the development of new protocols to communicate between players.
+
+* Provide a way for players to programmaticaly identify communication channels.
+
+
+## Full Example
+
+```json
+{
+  "api": {
+    "version": "draft",
+  },
+
+  "channels": {
+    "alliance": {
+      "segments": [43,87],
+    },
+    "needs": {
+      "protocol": "market",
+      "segments": [48],
+      "x-type": "sell"
+    },
+    "overflow": {
+      "protocol": "market",
+      "segments": [25],
+      "x-type": "buy"
+    },
+    "terminals": {
+      "protocol": "terminals",
+      "data": "[\"E34N53\", \"E65N13\" \"W52N14\"]"
+    },
+  }
+}
+```
+
+## Top Level Namespaces
+
+### api
+
+The `api` object contains metadata specifically about this standard. It can be used to mark version changes for compatibility purposes.
+
+
+### channels
+
+The `channels` object contains all the information needed to see which protocols a players is supporting and how to access them.
+
+* Each channel is given a name as its key in the `channels` object which points to an object that stores the channel metadata.
+
+* If there is no `protocol` field set then the channel name is used as the protocol.
+
+* If the `segments` field is defined it should contain an array of segment IDs. To retrieve the message for the channel each segment should be combined together in the order defined by this array. If this field is used the `data` field should not be used.
```

Comment:
I would add that segments should be concatenated in the order they're listed here, and not in numerical order.